src/core.c/Num.pm6

```diff
@@ -543,6 +566,26 @@ multi sub log(num $x --> num) {
     nqp::log_n($x);
 }
 
+multi sub modf(num $x is copy, num $places? --> List) { # sub source of the return values, patterned after sub log
```

Comment:
Why is this not simply deferring to the `.modf` version?

Reply:
Because I was blindly following the example of the log example.

Reply:
In that regard, is it better to implement a sub in terms of the better or vice versa?

In my own modules I have generally (if not always) defined the sub and used it in the corresponding class method.